nested/redis.json

```diff
@@ -18,7 +18,7 @@
     "resources": [
         {
             "type": "Microsoft.Cache/Redis",
-            "apiVersion": "2016-04-01",
+            "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.